### ISSUE TITLE
https - update any http references to https

### ISSUE THREAD

**Issue by smalers** [OpenWaterFoundation/owf-infomapper-poudre#162]
We have updated the `poudre.openwaterfoundation.org` website to `https`.  This causes problems, for example with drought monitor map that is using `http` URL to our S3 storage.  Need to update any such references to use the CloudFront `https` address.

Links to other websites as data sources and documentation that use `http` is probably OK since they are simple links that open in other tabs.

**Comment by smalers** [comment]
I updated the drought monitor URL to use `https` and now there is a CORS error.  See the following image.  Need to configure the websites to allow sharing data.

![image](https://user-images.githubusercontent.com/6448274/148924458-787da112-532b-43b4-89e6-c34cb5c0a027.png)

**Comment by smalers** [comment]
I updated all Markdown, etc. to use `https` when the links support, such as the OWF sites.

The CORS issue has been resolved by fixing the CloudFront configuration for the `data` site.  Additional changes will be implemented to use the `data` website to publish more datasets, rather than processing in the Poudre Basin Information workflows.  This will allow sharing of datasets and streamline the Poudre website.

Close this issue.
